packages/cli/src/command-helpers/ipfs.ts

```diff
@@ -1,3 +1,47 @@
-const DEFAULT_IPFS_URL = 'https://api.thegraph.com/ipfs/api/v0' as const;
+const DEFAULT_IPFS_URL = 'https://ipfs.thegraph.com/ipfs/api/v0' as const;
```

Comment:
The IPFS URL should just be https://ipfs.thegraph.com

For GET requests (read a file), you'd append `/ipfs/{cid}`.

For POST requests (adding a file) you'd append `/api/v0/add`